editor/enable-tracking-prompt/test/index.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { mount } from 'enzyme';
```

Comment:
In this case I think we could have used `shallow` rendering which tends to be more performant (doesn't mount into DOM, doesn't render descendants).